src/test/kotlin/blackjack/FixedDeck.kt

```diff
@@ -0,0 +1,22 @@
+package blackjack
+
+import blackjack.domain.Card
+import blackjack.domain.CardMark
+import blackjack.domain.CardNumber
+import blackjack.domain.Deck
+
+class FixedDeck : Deck {
+    override fun draw(): Card? {
+        return CACHED_CARDS.removeLastOrNull()
+    }
+
+    companion object {
+        private val CACHED_CARDS =
+            mutableListOf(
+                Card(CardNumber.ACE, CardMark.HEART),
+                Card(CardNumber.TEN, CardMark.HEART),
+                Card(CardNumber.ACE, CardMark.HEART),
+                Card(CardNumber.TEN, CardMark.HEART),
+            )
+    }
+}
```

Comment:
functional interface를 활용하면 아래와 같이 만들 수도 있겠군요!

장단점이 있으니 반영 여부는 석호님이 판단해주세요 🙂 

```suggestion
private val CACHED_CARDS =
    mutableListOf(
        Card(CardNumber.ACE, CardMark.HEART),
        Card(CardNumber.TEN, CardMark.HEART),
        Card(CardNumber.ACE, CardMark.HEART),
        Card(CardNumber.TEN, CardMark.HEART),
    )
    
fun fixedDeck() = Deck {
    CACHED_CARDS.removeLastOrNull()
}
```